opentelemetry-api/src/opentelemetry/context/propagation/__init__.py

```diff
@@ -11,8 +11,45 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import typing
+
+from opentelemetry.context import Context
 
 from .binaryformat import BinaryFormat
-from .httptextformat import HTTPTextFormat
+from .httptextformat import (
+    ContextT,
+    DefaultHTTPExtractor,
+    DefaultHTTPInjector,
+    Getter,
+    HTTPExtractor,
+    HTTPInjector,
+    Setter,
+)
```

Comment:
Is the goal here to change these classes' import paths?

For example, I see `opentelemetry-api/src/opentelemetry/propagation/__init__.py ` does:
```python
from opentelemetry.context.propagation import DefaultHTTPExtractor
```
instead of
```python
from opentelemetry.context.propagation.httptextformat import DefaultHTTPExtractor
```

Why do this?